### PR TITLE
Changelog for 1.12.4 release

### PR DESCRIPTION
**Pull request by techknowlogick**

### CODE REVIEW

**Comment by techknowlogick** [comment]
Blocked by backport of https://github.com/go-gitea/gitea/pull/12686

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,29 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.12.4](https://github.com/go-gitea/gitea/releases/tag/v1.12.4) - 2020-09-02
+
+* SECURITY
+  * Escape provider name in oauth2 provider redirect (#12648) (#12650)
+  * Escape Email on password reset page (#12610) (#12612)
```
Comment (techknowlogick) [inline CHANGELOG.md:11]:
```suggestion
  * Escape Email on password reset page (#12610) (#12612)
  * When reading expired sessions - expire them (#12686) (#12690)
```

**Comment by 6543** [comment]
Would be nice if we culd add https://github.com/go-gitea/gitea/pull/12652 too

**Comment by 6543** [comment]
And i think https://github.com/go-gitea/gitea/pull/12651 is bloking it :(

**Comment by 6543** [comment]
☝️ I can create a backport, eaven if it's not merged upstream?

**Comment by zeripath** [comment]
Can't backport a migration with our current migration architecture.

**Comment by jolheiser** [comment]
Reminder to update the version in the docs.  
Not really important on this PR, but the frontport will need it.

**Comment by 6543** [comment]
https://github.com/go-gitea/gitea/pull/12692 <- the backport ...

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,30 @@ This changelog goes through all the changes that have been made in each release
...
+  * When reading expired sessions - expire them (#12686) (#12690)
+* BUGFIXES
```
Comment (techknowlogick) [inline CHANGELOG.md:15]:
```suggestion
* ENHANCEMENTS
  * StaticRootPath configurable at compile time (#12371) (#12652)
* BUGFIXES
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,32 @@ This changelog goes through all the changes that have been made in each release
...
+## [1.12.4](https://github.com/go-gitea/gitea/releases/tag/v1.12.4) - 2020-09-02
+
+* SECURITY
+  * Escape provider name in oauth2 provider redirect (#12648) (#12650)
+  * Escape Email on password reset page (#12610) (#12612)
+  * When reading expired sessions - expire them (#12686) (#12690)
+* ENHANCEMENTS
+  * StaticRootPath configurable at compile time (#12371) (#12652)
+* BUGFIXES
+  * Expire time acknowledged for cache (#12605) (#12611)
```
Comment (techknowlogick) [inline CHANGELOG.md:17]:
```suggestion
  * Fix to show an issue that is related to a deleted issue (#12651) (#12692)
  * Expire time acknowledged for cache (#12605) (#12611)
```

**Comment by lafriks** [comment]
Is 1.14.8 already available for our CI?

**Review by 6543 — APPROVED** [review]

**Comment by techknowlogick** [comment]
@lafriks Yup

**Review by zeripath — APPROVED** [review]

**Comment by 6543** [comment]
ping lgtm bot

**Pull request merged**